xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-requiredrights/xwiki-platform-security-requiredrights-api/src/main/java/org/xwiki/platform/security/requiredrights/RequiredRight.java

```diff
@@ -58,6 +58,12 @@ public class RequiredRight
      */
     public static final RequiredRight MAYBE_SCRIPT = new RequiredRight(Right.SCRIPT, EntityType.DOCUMENT, true);
 
+    /**
+     * Admin right required.
+     * @since 15.10RC1
+     */
+    public static final RequiredRight ADMIN = new RequiredRight(Right.ADMIN, EntityType.DOCUMENT, false);
```

Comment:
Entity type document is wrong, translations with wiki scope need admin right on the wiki level. I've just added such a constant: https://github.com/xwiki/xwiki-platform/commit/070245b4c334d6887e062383782c6d1c50b64dcf#diff-671ca9e2172267eb16cafa05fe5bec12333592c6531c273f9378e046cec7a142R62-R67